src/ristretto/constants.rs

```diff
@@ -4,13 +4,15 @@
 //! Constant [NUMS](https://tools.ietf.org/id/draft-black-numscurves-02.html) points for the Ristretto curve. There are 10 provided, but this library currently only
```

Comment:
JFY, this link is broken. Can we update the reference please ?

Reply:
Good catch! I had not tested the link to ensure it was still active.

Reply:
Fixed.